GameServerLib/API/ApiFunctionManager.cs

```diff
@@ -133,21 +130,15 @@ public static void AddBuff(string buffName, float duration, byte stacks, BuffTyp
             onto.AddBuff(buff);
 
             _game.PacketNotifier.NotifyAddBuff(buff);
-
-            // HACK: Remove timers and use proper tick system to handle the buffs using priority queue of duration and order
-            CreateTimer(duration, () =>
-            {
-                RemoveBuff(buff);
-            });
         }
 
-        public static void AddBuffGameScript(string buffName, byte stacks, ISpell ownerSpell, BuffType buffType, IObjAiBase target, float duration, bool isUnique = false)
+        public static void AddBuffGameScript(string buffClass, string buffNamespace, byte stacks, ISpell ownerSpell, BuffType buffType, IObjAiBase target, float duration, bool isUnique = false)
```

Comment:
I'm not sure, but maybe it breaks some scripts.

Reply:
No it's used nowhere, it just to comply with the ObjAIBase function `void AddBuffGameScript(string buffNamespace, string buffClass, ISpell ownerSpell, float removeAfter = -1f, bool isUnique = false);`

Reply:
I mark this as resolved.